doc/proxy-clients.rst

```diff
@@ -28,6 +28,8 @@ You also need a `PSR-7 message implementation`_. If you use Guzzle 6, Guzzle’s
 implementation is already included. If you use another client, install one of
 the implementations. Recommended:
 
+TODO: i think we need the matching adapter for the factory, if guzzle6-adapter is not installed
```

Comment:
I’m not sure I follow:`php-http/guzzle6-adapter` installation is mentioned on line 25.